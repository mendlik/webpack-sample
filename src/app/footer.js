import style from './footer.scss';

const footer = () => {
  const element = document.createElement('footer');
  const text = document.createTextNode('Footer');
  element.appendChild(text);
  element.classList.add(style.content);
  return element;
};

export default footer;
