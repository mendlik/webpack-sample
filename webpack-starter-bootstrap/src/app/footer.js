import html from './footer.html';
import { parseHtmlNode } from './util';

const footer = () =>
  parseHtmlNode(html);

export default footer;
