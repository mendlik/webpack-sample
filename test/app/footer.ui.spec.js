import footer from '../../src/app/footer';

describe('Footer', () => {
  it('should create footer element', () => {
    const element = footer();
    expect(element).toMatchSnapshot();
  });
});
