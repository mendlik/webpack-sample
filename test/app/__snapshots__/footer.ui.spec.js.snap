// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should create footer element 1`] = `
<footer
  class="undefined"
>
  Footer
</footer>
`;
